src/Interface/Modules/Render/ES/SRInterface.cc

```diff
@@ -1221,6 +1221,16 @@ namespace SCIRun {
       mCore.renormalize(true);
     }
 
+    //----------------------------------------------------------------------------------------------
+    bool SRInterface::hasObject(std::string object)
+    {
+      for (auto it = mSRObjects.begin(); it != mSRObjects.end(); ++it)
```

Comment:
If you want, the more modern way to write this is:
`return std::find_if(mSRObjects.begin(), mSRObjects.end(), [&object](const Object& obj) { return obj.mName == object; }) != mSRObjects.end();`
where `Object` is the type of element in `mSRObjects` (which I forgot)